### ISSUE TITLE
Function `sanitize` is not static

### ISSUE THREAD

**Issue by masetto** [bleutzinn/grav-plugin-add-page-by-form#62]
Hi @bleutzinn 
I have this error when I call static method `slug`:
```
 grav.CRITICAL: Non-static method Grav\Plugin\AddPageByFormPlugin::sanitize() cannot be called statically
```

because the function `sanitize` is not declared as "static".

**Comment by bleutzinn** [comment]
Hello Samuele, I assume you are using PHP version 8.x ? If so the problem is as described in https://php.watch/versions/8.0/non-static-static-call-fatal-error
This plugin has not been tested with PHP 8 but it should by now. I'll look into it

**Comment by bleutzinn** [comment]
Could you please test with the updated script "add-page-by-form.php"? If all is well I'll create a new release.

**Comment by FREDderDRITTE** [comment]
Hey @bleutzinn

I encountered the same issue as @masetto using PHP version 8.0. Your updated script "add-page-by-form.php" fixed it for me.
Thx

**Comment by bleutzinn** [comment]
Thanks @FREDderDRITTE for the confirmation.

**Comment by bleutzinn** [comment]
Fixed in release v3.1.0.
